code/tailer/datastripe.q

```diff
@@ -154,14 +154,26 @@ savetables:{[dir;tablename]
 savealltables:{[dir]
     /- function takes the tailer hdb directory handle and a timestamp
     /- saves each table up to given period to their respective partitions
-    savetables[dir;]each .wdb.tablelist[];
-	
+    /- totals calculates the row count of tables in .wdb.tablelist[]
+    totals:{count value x}each .wdb.tablelist[];
+    /- log and return from function early if no table has crossed threshold
+    if[all totals<.wdb.numrows;
```

Comment:
As the savealltables function is used by both eod and eop, this condition will cause issues at eop and eod where the save actually might not occur because of it. I think this needs to be seperated into its own function outside of savealltables so that the functionality is just for saving and nothing else.